main_timer_rate: 1.0 # [Hz]
uav_state_timer_rate: 100.0 # [Hz]
